packages/grafana-ui/src/components/Card/CardContainer.tsx

```diff
@@ -93,7 +97,7 @@ export const getCardContainerStyles = (
       display: 'grid',
       position: 'relative',
       gridTemplateColumns: 'auto 1fr auto',
-      gridTemplateRows: 'auto auto 1fr auto',
+      gridTemplateRows: hasDescriptionComponent ? 'auto auto 1fr auto' : '1fr auto auto auto',
```

Comment:
What do you think of the idea of clarifying the intention of `hasDescriptionComponent` by expressed the combination of `gridTemplateAreas`, `gridTemplateRows` and `gridTemplateColumns` through `gridTemplate`.

Since those three css properties can be expressed as `grid-template` like this:
```css
grid-template: "Figure Heading Tags"
"Figure Meta Tags"
"Figure Description Tags" 1fr
"Figure Actions Secondary" / auto 1fr auto
```

We could express our ternary as:
```tsx
gridTemplate: 
  hasDescriptionComponent ? `
    "Figure Heading Tags"
    "Figure Meta Tags"
    "Figure Description Tags" 1fr
    "Figure Actions Secondary" / auto 1fr auto
  ` : 
  `
    "Figure Heading Tags" 1fr
    "Figure Meta Tags"
    "Figure Actions Secondary" / auto 1fr auto
  `,
```